testapp/src/main/java/org/schabi/testapp/MainActivity.kt

```diff
@@ -27,6 +28,7 @@ class MainActivity : AppCompatActivity() {
                 action = ACTION_START
             }
             startService(serviceIntent);
+            DeviceList.getOnwDeviceInfo(this)
```

Comment:
Why is it needed to always run this on startup? Can't the database be lazy-loaded when needed?